cve_bin_tool/checkers/sqlite.py

```diff
@@ -47,7 +45,7 @@ def get_version_map():
                     version_map.append([last_version, id_match.group(1)])
                     break
 
-    except error.URLError as err:
+    except requests.ConnectionError as err:
```

Comment:
I'm not entirely sure about replacing `error.URLError` with `requests.ConnectionError`.